.github/workflows/CI_build.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push, pull_request]
```

Comment:
Why on these? We don't usually need to track changes here, it is all about the upstream repo. Is there a way to trigger from boostorg/boost changes to master?

If not (or if that is running too much) maybe run daily?

Reply:
Not sure if it could be easily triggered by commits to boosts github without having the event within the boost github repo and also https://github.com/marketplace/actions/track-and-record-changes-in-other-repos seems to be quite complicated.
So I started with adding a daily build of the master snapshot. An issue seems to be the switch from e.g. 1.83.0 -> 1.84.0 which needs to be done manually.

Reply:
I've made the change to 1.84 in the mainline branch.